client/src/components/SignUp.js

```diff
@@ -1,41 +1,96 @@
 import React, { Component } from "react";
+import { Alert } from 'reactstrap';
+
 export default class SignUp extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            fName: "",
+            lName: "",
+            email: "",
+            confirmEmail: "",
+            password: "",
+            confirmPassword: "",
+            renderAlert: false,
+        };
+    }
+
+    handleFieldChange = (e) => {
+        this.setState({[e.target.name] : e.target.value, renderAlert: false});
```

Comment:
I would put the renderAlert: false on a separate line so that we can easily see that you are updating two things in the state each time a field is changed.